README.md

```diff
@@ -1,25 +1,103 @@
+### 1. **How do I start editing the recipe?**
 
+There are two ways to get started:
 
-Build status
-------------
+a. If it is a python package you can generate a skeleton as a starting point with
+`conda skeleton pypi your_package_name`. You do *not* have to use skeleton, and the
+recipes produced by skeleton will need to be edited.
 
-[![Circle CI](https://circleci.com/gh/conda-forge/staged-recipes/tree/master.svg?style=shield)](https://circleci.com/gh/conda-forge/staged-recipes/tree/master)
+b. Look at one of [these examples](https://github.com/conda-forge/staged-recipes/tree/master/recipes)
+in this repository and modify it as necessary.
 
-[![Build Status](https://travis-ci.org/conda-forge/staged-recipes.svg?branch=master)](https://travis-ci.org/conda-forge/staged-recipes)
+Your final recipe should have no comments and follow the order in the example.
 
-[![Build status](https://ci.appveyor.com/api/projects/status/3lju80dibkmowsj5/branch/master?svg=true)](https://ci.appveyor.com/project/conda-forge/staged-recipes/branch/master)
+*If there are details you are not sure about please open a pull request. The conda-forge team will be happy to answer your questions.*
+
+### 2. **How do I populate the `hash` field?**
+
+If your package is on PyPI, you can get the md5 hash from your package's page on PyPI; look for the `md5` link next to the download link for your package.
+
+You can also generate a hash from the command line on Linux (and Mac if you install the necessary tools below). If you go this route, the `sha256` hash is preferable to the `md5` hash.
+
+To generate the `md5` hash: `md5 your_sdist.tar.gz`
+
+To generate the `sha256` hash: `openssl sha256 your_sdist.tar.gz`
+
+You may need the openssl package, available on conda-forge
+`conda install openssl -c conda-forge`
+
+### 3. **How do I exclude a platform?**
+
+Use the `skip` key in the `build` section along with a selector:
+
+```yaml
+build:
+    skip: true  # [win]
+```
 
+A full description of selectors is [in the conda docs](http://conda.pydata.org/docs/building/meta-yaml.html#preprocessing-selectors).
+
+### 4. **What does `numpy x.x` mean?**
+
+If you have a package which links against numpy you need to build and run against the same version of numpy.
+Putting `numpy x.x` in the build and run requirements ensure that a separate package will be built for each
+version of numpy that conda-forge builds against.
+
+### 5. **What does the `build: 0` entry mean?**
+
+The build number is used when the source code for the package has not changed but you need to make a new
+build. For example, if one of the dependencies of the package was not properly specified the first time
+you build a package, then when you fix the dependency and rebuild the package you should increase the build
+number.
+
+When the package version changes you should reset the build number to `0`.
+
+### 6. **Do I have to import all of my unit tests into the recipe's `test` field?**
+
+No, you do not.
```

Comment:
We basically have people run the test suite if it is compiled code. Pure Python stuff we have let slide. Though there have occasionally been discussions about requiring the test suite there too. At present, it is only compiled code that needs this.

> although because of the way `make test` works, it has to go in the build script, not the test field

Yeah, that's a bit inelegant. I have seen hacky ways around this, but am a bit uncomfortable with how well or whether they will work in the long run. Perhaps there could be some discussion with upstream about how to improve on this. Personally am not invested in this one though.